### DISCUSSION TITLE
Justified Text

### DISCUSSION THREAD

**Discussion by thewhaleiam** [awcodes/filament-tiptap-editor#411] [Q&A] [answered]
How would I go about implementing the option to justify text? I can see that text alignment options such as left, right, and center are available out of the box, but it appears that justifying text is unavailable for formatting. Thanks in advance.

**Comment by awcodes** [comment, marked as answer]
add 'align-justify' to the profile in the config.

**Reply by thewhaleiam** [reply]
Thanks I didn't see that option
